src/main.server.ts

```diff
@@ -0,0 +1,11 @@
+import { enableProdMode } from '@angular/core';
+import { environment } from './environments/environment';
+
+if (environment.production) {
+  enableProdMode();
+}
+
+export { AppServerModule } from './app/app.server.module';
```

Comment:
@CryceTruly  what do you think of having these imports at the top of the file, before the `if condition` 
I think it can be great for uniformity

Reply:
@CryceTruly , I've seen this are exports. so its totally fine 👍

Reply:
Thanks, @Tevinthuku, I have worked on this.